src/module.c

```diff
@@ -876,6 +876,18 @@ int64_t commandFlagsFromString(char *s) {
  *                     to authenticate a client. 
  * * **"may-replicate"**: This command may generate replication traffic, even
  *                        though it's not a write command.  
+ *
+ * The last three parameters specify which arguments of the new command are
+ * Redis keys.
+ *
+ * * 'firstkey': One-based index of the first argument that's a key.
```

Comment:
i'm not sure stating that it's one-based is correct.
it actually refers to the argv array, in which [0] is the command itself.
maybe we should refer to https://redis.io/commands/command ?